challenge.md

```diff
@@ -0,0 +1,58 @@
+# Bowling game 
+
```

Comment:
I really like how you explain the thought process of planning. It does help me to understand how to follow the code.